### DISCUSSION TITLE
Bug ?

### DISCUSSION THREAD

**Discussion by Julien4135** [robske110/IDDataLogger#48] [Q&A] [answered]
Hello, my car went to the garage they acted on the system to redo a program and since then I am no longer connected to my car.

My car is online though  

![image](https://user-images.githubusercontent.com/93113346/153921836-764ffa4b-d3c5-4e6d-bcf7-aa9832032b4f.png)

**Comment by JimKnopf1503** [comment, marked as answer]
Hi!
For me, IDDatalogger is still working.
You say, you can connect using WeConnect-ID and have actual data?
I see, in the errormessage "could not find credentialForm". VW made a change in the login procedure. You may need to update IDDatalogger. (git pull && git submodule update)
Burkhard

**Reply by Julien4135** [reply]
I don't know what has changed but I updated and it's good everything works again as before I haven't lost anything as history it's great!
Thank you very much @JimKnopf1503
